s64da_benchmark_toolkit/prepare.py

```diff
@@ -122,7 +122,7 @@ def _apply_schema_modifications(self, schema_sql):
             # No S64 DA, no modifications
             return schema_sql
 
-        if not (version.major <= 4 and version.minor < 1):
+        if not (version.major < 4 or version.major <= 4 and version.minor < 1):
```

Comment:
If we use more such comparisons we might want to introduce a `version_compare` function (not in this PR though).